tarorpc/taro.proto

```diff
@@ -22,6 +22,12 @@ service Taro {
     */
     rpc ListUtxos (ListUtxosRequest) returns (ListUtxosResponse);
 
+    /* tarocli: `assets groups`
+    ListGroups lists the asset groups known to the target daemon, and the assets
+    held in each group.
+    */
+    rpc ListGroups (ListGroupsRequest) returns (ListGroupsResponse);
```

Comment:
Ah ok I see the usage here, would be useful to get a bit more context on how this is used. 

I'm guessing the new call which use some identifier listed here to allow further issuance into an existing asset group?

Reply:
This is example output, from the collectible send itest:

```
"groups": {
                "0243a776990c748ce93887a6fcb683dafca79707270f90c43ce7564b062743f7b0": {
                    "assets_with_group": [
                        {
                            "id": "2606c42f5bd0874ffbae23cdbf8261315f298a5dbdcf276b6e13bc8a1044ab4c",
                            "amount": "0",
                            "lock_time": 0,
                            "relative_lock_time": 0,
                            "tag": "itestbuxx-collectible-brrr",
                            "meta_data": "736f6d65206d65746164617461",
                            "type": "COLLECTIBLE",
                            "group_key": "0243a776990c748ce93887a6fcb683dafca79707270f90c43ce7564b062743f7b0"
                        },
                        {
                            "id": "2606c42f5bd0874ffbae23cdbf8261315f298a5dbdcf276b6e13bc8a1044ab4c",
                            "amount": "1",
                            "lock_time": 0,
                            "relative_lock_time": 0,
                            "tag": "itestbuxx-collectible-brrr",
                            "meta_data": "736f6d65206d65746164617461",
                            "type": "COLLECTIBLE",
                            "group_key": "0243a776990c748ce93887a6fcb683dafca79707270f90c43ce7564b062743f7b0"
                        }
                    ]
                }
            }
```

The example user flow is, user already issued an asset with a emission enabled, now wants to reissue. They call `list groups` and find the asset and corresponding group key. They copy the group key shown and paste when building their `mint` command to perform the reissuance.

Reply:
I think `"assets_with_group"` is redundant here. Can just be `"assets"`.